### DISCUSSION TITLE
2023-09-11

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#486] [daily_meeting]
### 오늘의 할 일
- 프론트 2~3시 회의
- SSE 관련 회의 (4시~)

### 개인 일정
- 노아 3시 근로회의

### 내일의 할 일
- 진행 도중의 next-step 관련 SSE 내용 정의

**Comment by aak2075** [comment]
### 체크인

9점 - 어제 운동을 못한게 아쉬워요

### 체크아웃

10점 - 오늘 하루도 끝

**Comment by woosung1223** [comment]
### 체크인
9점 - 월요일이지만 주말동안 좀 쉰 것 같아서 컨디션이 낫배드 하네요

### 체크아웃
8점 - SSE 회의가 생각보다 오래걸리네요 ㅠ

**Comment by jaehee329** [comment]
### 체크인
8점 - 잘 쉬었습니다

### 체크아웃
8점 - 이것이 월요일의 맛

**Comment by yeopto** [comment]
### 체크인

10점 - 에러핸들링 끝이보입니다.

### 체크아웃

10점 - 오늘 하루 금방 지나갔네요

**Comment by woo-jk** [comment]
### 체크인
10점 -오늘 컨디션 좋네요~ 

### 체크아웃
8점 - 머리가 어지럽다.

**Comment by nlom0218** [comment]
9점 - 머리가 아파요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 주말에 늘어져서 쉰 것 같으면서도 할 건 많아서 상쇄되는 느낌이네요. 얼른 프젝진행 해야될 것 같아요...

### 체크아웃
8점 - 오랜만에 프젝 얘기 하니까 재밌으면서도 언제 구현하지라는 조급함이 드네요... 화이팅해봅시다!
